src/shared/scrapers/BRA/county-level.js

```diff
@@ -0,0 +1,5621 @@
+    'Nova Glória, GO',
+    'Nova Iguaçu de Goiás, GO',
+    'Nova Roma, GO',
+    'Nova Veneza, GO',
+    'Novo Brasil, GO',
+    'Novo Gama, GO',
+    'Novo Planalto, GO',
+    'Orizona, GO',
+    'Ouro Verde de Goiás, GO',
+    'Ouvidor, GO',
+    'Padre Bernardo, GO',
+    'Palestina de Goiás, GO',
+    'Palmeiras de Goiás, GO',
+    'Palmelo, GO',
+    'Palminópolis, GO',
+    'Panamá, GO',
+    'Paranaiguara, GO',
+    'Paraúna, GO',
+    'Perolândia, GO',
+    'Petrolina de Goiás, GO',
+    'Pilar de Goiás, GO',
+    'Piracanjuba, GO',
+    'Piranhas, GO',
+    'Pirenópolis, GO',
+    'Pires do Rio, GO',
+    'Planaltina, GO',
+    'Pontalina, GO',
+    'Porangatu, GO',
+    'Porteirão, GO',
+    'Portelândia, GO',
+    'Posse, GO',
+    'Professor Jamil, GO',
+    'Quirinópolis, GO',
+    'Rialma, GO',
+    'Rianápolis, GO',
+    'Rio Quente, GO',
+    'Rio Verde, GO',
+    'Rubiataba, GO',
+    'Sanclerlândia, GO',
+    'Santa Bárbara de Goiás, GO',
+    'Santa Cruz de Goiás, GO',
+    'Santa Fé de Goiás, GO',
+    'Santa Helena de Goiás, GO',
+    'Santa Isabel, GO',
+    'Santa Rita do Araguaia, GO',
+    'Santa Rita do Novo Destino, GO',
+    'Santa Rosa de Goiás, GO',
+    'Santa Tereza de Goiás, GO',
+    'Santa Terezinha de Goiás, GO',
+    'Santo Antônio da Barra, GO',
+    'Santo Antônio de Goiás, GO',
+    'Santo Antônio do Descoberto, GO',
+    'São Domingos, GO',
+    'São Francisco de Goiás, GO',
+    "São João d'Aliança, GO",
+    'São João da Paraúna, GO',
+    'São Luís de Montes Belos, GO',
+    'São Luiz do Norte, GO',
+    'São Miguel do Araguaia, GO',
+    'São Miguel do Passa Quatro, GO',
+    'São Patrício, GO',
+    'São Simão, GO',
+    'Senador Canedo, GO',
+    'Serranópolis, GO',
+    'Silvânia, GO',
+    'Simolândia, GO',
+    "Sítio d'Abadia, GO",
+    'Taquaral de Goiás, GO',
+    'Teresina de Goiás, GO',
+    'Terezópolis de Goiás, GO',
+    'Três Ranchos, GO',
+    'Trindade, GO',
+    'Trombas, GO',
+    'Turvânia, GO',
+    'Turvelândia, GO',
+    'Uirapuru, GO',
+    'Uruaçu, GO',
+    'Uruana, GO',
+    'Urutaí, GO',
+    'Valparaíso de Goiás, GO',
+    'Varjão, GO',
+    'Vianópolis, GO',
+    'Vicentinópolis, GO',
+    'Vila Boa, GO',
+    'Vila Propício, GO',
+    'Brasília, DF'
+  ],
+
+  async scraper() {
+    const data = await fetch.csv(this.url, false);
+    const scrapeDate = process.env.SCRAPE_DATE ? datetime.getYYYYMMDD(process.env.SCRAPE_DATE) : datetime.getYYYYMMDD();
+    let latestDate = data[0].date;
+
+    if (datetime.dateIsBefore(latestDate, scrapeDate)) {
+      console.error('  🚨 Timeseries for BRA has not been updated, using %s instead of %s', latestDate, scrapeDate);
+    } else {
+      latestDate = datetime.getYYYYMMDD(scrapeDate);
+    }
+
+    let counties = data
+      .filter(row => {
+        return row.date === latestDate;
+      })
+      .map(row => {
+        return {
+          cases: parse.number(row.confirmed) || 0,
+          deaths: parse.number(row.deaths) || 0,
```

Comment:
If the number is undefined, don't return 0, we should be leaving the field undefined.

Reply:
Done!